spring-integration-r2dbc/src/test/java/org/springframework/integration/r2dbc/outbound/R2dbcMessageHandlerTests.java

```diff
@@ -29,79 +27,49 @@
 import org.junit.Assert;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.mockito.Answers;
 
-import org.springframework.beans.factory.BeanFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.FilterType;
-import org.springframework.data.r2dbc.config.AbstractR2dbcConfiguration;
+import org.springframework.context.annotation.Import;
 import org.springframework.data.r2dbc.core.DatabaseClient;
 import org.springframework.data.r2dbc.core.R2dbcEntityTemplate;
-import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories;
 import org.springframework.data.relational.core.query.Criteria;
-import org.springframework.data.repository.reactive.ReactiveCrudRepository;
 import org.springframework.integration.expression.FunctionExpression;
+import org.springframework.integration.r2dbc.config.R2dbcDatabaseConfiguration;
+import org.springframework.integration.r2dbc.entity.Person;
+import org.springframework.integration.r2dbc.repository.PersonRepository;
 import org.springframework.integration.support.MessageBuilder;
 import org.springframework.messaging.Message;
 import org.springframework.test.context.junit.jupiter.SpringJUnitConfig;
 
-import io.r2dbc.h2.H2ConnectionConfiguration;
-import io.r2dbc.h2.H2ConnectionFactory;
-import io.r2dbc.spi.ConnectionFactory;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Hooks;
 import reactor.core.publisher.Mono;
 import reactor.test.StepVerifier;
 
 /**
- *  @author Rohan Mukesh
+ * @author Rohan Mukesh
  *
- *  @since 5.4
+ * @since 5.4
  */
 @SpringJUnitConfig
```

Comment:
The `@DirtiesContext` here as well to close an application context and shoultdown embedded DB altogether.